cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,39 @@
+class TimeLoggingPage {
+  visit() {
+    cy.visit('/time-logging');
+  }
+
+  getUrl() {
+    return cy.url();
+  }
+
+  getPageTitle() {
+    return cy.get('.page__title');
+  }
+
+  getCalendar() {
+    return cy.get('.calendar');
+  }
+
+  getTileForm() {
+    return cy.get('.tile.form');
+  }
+
+  getUserInfoTitle() {
+    return cy.get('.user-info__title');
+  }
+
+  getMainNav() {
+    return cy.get('.main-nav');
+  }
+
+  getSelectedDateToday() {
+    return cy.get('.calendar--today.calendar--selected');
+  }
+
+  getMainNavActiveLink() {
```

Comment:
I suppose this refers to the active selected tab? If I'm correct, why not call it something like 'getActiveTab()'? It will make your life easier when you see this function somewhere later on :)

Reply:
Fixed :)